var/spack/repos/builtin/packages/adol-c/package.py

```diff
@@ -0,0 +1,69 @@
+from spack import *
+import sys
+
+class AdolC(Package):
+    """A package for the automatic differentiation of first and higher derivatives of vector functions in C and C++ programs by operator overloading."""
+    homepage = "https://projects.coin-or.org/ADOL-C"
+    url      = "http://www.coin-or.org/download/source/ADOL-C/ADOL-C-2.6.1.tgz"
+
+    version('head',  svn='https://projects.coin-or.org/svn/ADOL-C/trunk/')
+    version('2.6.1', '1032b28427d6e399af4610e78c0f087b')
+    
+    variant('doc',      default=True,  description='Install documentation')
+    variant('openmp',   default=False, description='Enable OpenMP support')
+    variant('sparse',   default=False, description='Enable sparse drivers')
+    variant('tests',    default=True,  description='Build all included examples as a test case')
+
+    def install(self, spec, prefix):
+        make_args = ['--prefix=%s' % prefix]
+
+        # --with-cflags=FLAGS     use CFLAGS=FLAGS (default: -O3 -Wall -ansi)
+        # --with-cxxflags=FLAGS   use CXXFLAGS=FLAGS (default: -O3 -Wall)
+
+        if '+openmp' in spec:
+            make_args.extend([
+                '--with-openmp-flag=-fopenmp' # FIXME: Is this required? -I <path to omp.h> -L <LLVM OpenMP library path>
+            ])
+
+        if '+sparse' in spec:
+            make_args.extend([
+                '--enable-sparse'
+            ])
+
+        # We can simply use the bundled examples to check
+        # whether Adol-C works as expected
+        if '+tests' in spec:
+            make_args.extend([
+                '--enable-docexa', # Documeted examples
+                '--enable-addexa'  # Additional examples
+            ])
+            if '+openmp' in spec:
+                make_args.extend([
+                    '--enable-parexa'  # Parallel examples
+                ])
+
+        configure(*make_args)
+        make()
+        make("install")
+        
+        # Copy the config.h file, as some packages might require it
+        source_directory = self.stage.source_path
+        config_h = join_path(source_directory,'ADOL-C/src/config.h')
+        install(config_h, join_path(prefix.include,'adolc'))
+        
+        # Install documentation to {prefix}/share
+        if '+doc' in spec:
+            install_tree('ADOL-C/doc',join_path(prefix.share,'doc'))
+        
+        # Install examples to {prefix}/share
+        if '+tests' in spec:
+            install_tree('ADOL-C/examples',join_path(prefix.share,'examples'))
+        
+            # Run some examples that don't require user input
+            # TODO: Check that bundled examples produce the correct results
+            with working_dir(join_path(source_directory,'ADOL-C/examples')):
+              Executable('tapeless_scalar')
```

Comment:
i am not sure, but this may not be running it. You need another `()` at the end. Same below.